avm/utilities/pipelines/e2eValidation/resourceDeployment/New-TemplateDeployment.ps1

```diff
@@ -155,7 +155,8 @@ function New-TemplateDeploymentInner {
         Write-Debug ('{0} entered' -f $MyInvocation.MyCommand)
 
         # Load helper
-        . (Join-Path (Get-Item -Path $PSScriptRoot).parent.parent.FullName 'sharedScripts' 'Get-ScopeOfTemplateFile.ps1')
+        $repoRoot = (Get-Item -Path $PSScriptRoot).parent.parent.parent.parent.parent.FullName
```

Comment:
Should we have the reporoot as input instead?

Reply:
done

Reply:
I think we've lost the implemented change with the latest pull. I can't see much in the diff anymore

Reply:
Say what? :D 
![Uploading image.png…]()

Reply:
Yes, this above is the only change I'm seeing now. Only adding reporoot as input. Everything else is gone, is that the only intention of this PR?

Reply:
No, you're right. The duck

Reply:
Should be recovered now @eriqua